database/seeders/DummyDataSeeder.php

```diff
@@ -20,7 +19,7 @@ public function run(): void
         $user = User::factory([
             "name" => "Admin",
             "email" => "admin@example.com",
-            "password" => Hash::make("password"),
+            "password" => "password",
```

Comment:
Why you would like to have nothashed pasword here?

Reply:
because we are hashing password in service when user are created and if we leave hashing here password is doubled encrypted and it's impossible to login

Reply:
I'm not sure if I like this solution here. We should understand `password` field as a place where we have always a hashed password. With the observer hashing passwords, we have two different approaches: sometimes it's plain password and sometimes it's hashed. Please move hashing into some service.